hugo/README.md

```diff
@@ -0,0 +1,9 @@
+## Hosting a [hugo](https://github.com/spf13/hugo) blog fronted by caddy
+
+This assumes only the following things :
+ - You have `caddy` installed and selected the `git` and `-hugo` plugins during the download
+ - You may or may not have `hugo` installed. If not, the `hugo` plugin will automatically fetch the latest version from the github releases of [hugo](https://github.com/spf13/hugo/releases)
+ 
+ 
+### Note : Caddy 0.8.3 is the recommended version until plugin authors update their middleware and caddyserver.com/download starts builds of 0.9
```

Comment:
I'm neither planning to die, nor go back to 1960's webservers so definitely
yes, I'll be around to update it before you remember it needs to be updated
:)

On Jul 18, 2016 8:15 PM, "Matt Holt" <notifications@github.com> wrote:

In hugo/README.md
<https://github.com/caddyserver/examples/pull/15#discussion_r71160246>:

> @@ -0,0 +1,9 @@
> +## Hosting a [hugo](https://github.com/spf13/hugo) blog fronted by caddy
> +
> +This assumes only the following things :
> + - You have `caddy` installed and selected the `git` and `-hugo` plugins during the download
> + - You may or may not have `hugo` installed. If not, the `hugo` plugin will automatically fetch the latest version from the github releases of [hugo](https://github.com/spf13/hugo/releases)
> +
> +
> +### Note : Caddy 0.8.3 is the recommended version until plugin authors update their middleware and caddyserver.com/download starts builds of 0.9

Be sure to update this after 0.9 comes out, if you could. :)

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/caddyserver/examples/pull/15/files/cbda6d7cca8175ce8e2497bac1c791a0e818b37e#r71160246>,
or mute the thread
<https://github.com/notifications/unsubscribe-auth/AMuuGrb6NqxM94rrUQhguXV__MBMaIeBks5qW5FigaJpZM4JNESK>
.